components/python/elixir/elixir-26.p5m

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
It was faster and easier to do it this way. And Elixir does not support Python 3.x. So in the future, if we deprecate Python 2.6, we just delete the file.